src/geometry/cube.rs

```diff
@@ -1,7 +1,52 @@
 //! An axis-aligned cube.
 
-use cgmath::{Point3, Vector3};
-use collision::{Aabb, Aabb3};
+use crate::math::sat::{self, ConvexPolyhedron, Intersector, Relation};
+use crate::math::PointCulling;
+use arrayvec::ArrayVec;
+use cgmath::{BaseFloat, Point3, Vector3};
+use collision::{Aabb, Contains};
+use num_traits::Bounded;
+
+pub use collision::Aabb3;
+
+impl<S> PointCulling<S> for Aabb3<S>
+where
+    S: 'static + BaseFloat + Sync + Send + Bounded,
+{
+    fn contains(&self, p: &Point3<S>) -> bool {
+        Contains::contains(self, p)
+    }
+
+    fn intersects_aabb3(&self, aabb: &Aabb3<S>) -> bool {
```

Comment:
Would it make sense to return `Relation` here?

Reply:
Yes, absolutely! That's planned for a future PR, I wanted to keep them small.

Reply:
Ok, great!